### DISCUSSION TITLE
<C-v> conflict with vsplit and paste

### DISCUSSION THREAD

**Discussion by Weissle** [nvim-tree/nvim-tree.lua#1330] [Q&A] [answered]
So I am using WSL and the terminal is Windows terminal. When I press <C-v>, its real meaning is paste but not split. I wonder how you guys avoid this? Change the shortcut of nvim-tree.lua or disable the paste function of Windows terminal?

**Comment by kyazdani42** [comment, marked as answer]
I don't use windows really, so i can't say, but you ought to disable the binding for `<C-v>` in nvim tree and bind its action to something else that you like.

**Reply by Weissle** [reply]
Thanks.
